go.mod

```diff
@@ -6,7 +6,7 @@ require (
 	github.com/Masterminds/semver v1.5.0
 	github.com/coreos/go-systemd v0.0.0-20190719114852-fd7a80b32e1f
 	github.com/gardener/etcd-druid v0.1.1-0.20200206100633-e7606e1225d7
-	github.com/gardener/gardener v1.1.0
+	github.com/gardener/gardener v1.1.1-0.20200306090858-1aa0c7eb14d1
```

Comment:
`make revendor` does minor change under vendor for me
```
git diff --shortstat
 2 files changed, 35 insertions(+), 35 deletions(-)
```

Is it the same for you?

Reply:
Hm, no. Why is that?

Reply:
Looks like some whitespace/line ending issue under vendor. However it is not a bit thing.